stable/cluster-lifecycle/values.yaml

```diff
@@ -24,6 +24,17 @@ clusterController:
       memory: "256Mi"
       cpu: "500m"
 
+ClusterCuratorController:
+  name: cluster-curator-controller
+  shortName: cluster-curator
+  resources:
+    limits:
+      cpu: "10m"
+      memory: "45Mi"
```

Comment:
worried about the limit, but as it only watches managedcluster, so should be enough